src/webui/www/private/scripts/client.js

```diff
@@ -623,11 +624,20 @@ window.addEventListener("DOMContentLoaded", function() {
 
         // Sort trackers by hostname
         const sortedList = [];
-        trackerList.forEach((tracker, hash) => sortedList.push({
-            trackerHost: getHost(tracker.url),
-            trackerHash: hash,
-            trackerCount: tracker.torrents.length
-        }));
+        trackerList.forEach(({ host, trackerTorrentMap }, hash) => {
+            const uniqueTorrents = new Set();
+            for (const torrents of trackerTorrentMap.values()) {
+                for (const torrent of torrents) {
+                    uniqueTorrents.add(torrent);
+                }
+            }
```

Comment:
`Set()` can take a iterable parameter so I suppose this would work:
```suggestion
            for (const torrents of trackerTorrentMap.values())
                uniqueTorrents.add(torrents);
```

Also in mocha-init.js

Reply:
oops... I mixed up with something else... disregard this.